t/10-jobs.t

```diff
@@ -488,6 +488,25 @@ subtest 'carry over, including soft-fails' => sub {
         is($inv->{test_log}, 'No test changes recorded, test regression unlikely', 'git log with no test changes');
     };
 
+    subtest 'investigation provides test_log with git stats' => sub {
+        path('t/data/last_good.json')->copy_to(path(($job->_previous_scenario_jobs)[1]->result_dir(), 'vars.json'));
+        path('t/data/first_bad.json')->copy_to(path($job->result_dir(),                               'vars.json'));
+        $job->done;
+        is($job->result, OpenQA::Jobs::Constants::FAILED, 'job result is failed');
+        ok(my $inv = $job->investigate, 'job can provide investigation details');
+        ok($inv,                        'job provides failure investigation');
```

Comment:
this is a verbatim copy of the lines in the former subtest. Can you combine both tests to not repeat this? You could nest another subtest within the above. If you really want to keep it then better extract a helper function but then avoid any Test::More commands in here and please use unique test description strings

Reply:
*this change is ready for review*

Reply:
Sorry, I don't see a difference